pkg/envoy/config/config.go

```diff
@@ -53,6 +54,7 @@ func (r ProxyConfig) Flags(flags *pflag.FlagSet) {
 	flags.Uint("proxy-connect-timeout", 2, "Time after which a TCP connect attempt is considered failed unless completed (in seconds)")
 	flags.Uint("proxy-initial-fetch-timeout", 30, "Time after which an xDS stream is considered timed out (in seconds)")
 	flags.Uint("proxy-gid", 1337, "Group ID for proxy control plane sockets.")
+	flags.Int64("proxy-max-active-downstream-connections", 50000, "Set Envoy HTTP option max_active_downstream_connections. Default 50000")
```

Comment:
No need to mention `Default 50000` in the description. Defaults are automatically documented based on the default value.